documentation/operators/src/nodes/network-requester-setup.md

```diff
@@ -14,10 +18,13 @@
 Make sure you do the preparation listed in the [preliminary steps page](../preliminary-steps.md) before setting up your network requester.
 
 ## Network Requester Whitelist
+
 If you have access to a server, you can run the network requester, which allows Nym users to send outbound requests from their local machine through the mixnet to a server, which then makes the request on their behalf, shielding them (and their metadata) from clearnet, untrusted and unknown infrastructure, such as email or message client servers.
 
 By default the network requester is **not** an open proxy (although it can be used as one). It uses a file called `allowed.list` (located in `~/.nym/service-providers/network-requester/<NETWORK-REQUESTER-ID>/`) as a whitelist for outbound requests.
 
+**Note:** If you run network requester as a part of the gateway the `allowed.list` will be stored in `~/.nym/gateways/snus/data/network-requester-data/allowed.list`.
```

Comment:
Looks like `snus` should be changed to `ID` ?

Reply:
lol, done